pkgs/by-name/ol/ollama/gpu-runner-targets.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/llama/Makefile b/llama/Makefile
+index 47a87a75..8b71c999 100644
+--- a/llama/Makefile
++++ b/llama/Makefile
+@@ -15,7 +15,7 @@ else ifeq ($(OS),linux)
+ 	HIP_LIB_DIR := $(shell ls -d $(HIP_PATH)/lib 2>/dev/null)
+ 	CUDA_PATH?=/usr/local/cuda
+ 	CUDA_11:=$(shell ls -d $(CUDA_PATH)-11 2>/dev/null)
+-	CUDA_12:=$(shell ls -d $(CUDA_PATH)-12 2>/dev/null)
++	CUDA_12:=$(shell ls -d $(CUDA_PATH) 2>/dev/null)
+ endif
```

Comment:
I've temporarily hardcoded it to CUDA 12 for verification purposes, and it works as expected. but I'm not sure of the best approach to support both CUDA 11 and 12. Any suggestions?